### PR TITLE
Rename everything: use scoped packages

### PR DESCRIPTION
**Pull request by hzoo**
Closes https://github.com/babel/babel/pull/5294 (rebased/fixed version of previous PR from like a year ago)

### CODE REVIEW

File: codemods/babel-plugin-codemod-optional-catch-binding/package.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
-  "name": "babel-plugin-codemod-optional-catch-binding",
+  "name" : "@babel/plugin-codemod-optional-catch-binding",
```
Comment (loganfsmyth) [inline codemods/babel-plugin-codemod-optional-catch-binding/package.json]: Got an extra space before the colon here.

File: experimental/babel-preset-env/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "babel-preset-env",
+  "name" : "@babel/preset-env",
```
Comment (loganfsmyth) [inline experimental/babel-preset-env/package.json]: Got an extra space before the colon here.

**Comment by xtuc** [comment]
Did you use a codemod for the switch to scoped?

I think it would be useful to publish it for plugin authors who'd like to switch easily.

**Comment by hzoo** [comment]
Alright it's fine to merge as is and do the rest ^.

I think we can figure out the directory change later

File: packages/babel-standalone/README.md [hunk 3]
```diff
@@ -16,11 +16,11 @@ It's true that using Babel through Webpack, Browserify or Gulp should be suffici
 Installation
 ============
 
-There are several ways to get a copy of babel-standalone. Pick whichever one you like:
+There are several ways to get a copy of @babel/standalone. Pick whichever one you like:
 
-- Use it via UNPKG: https://unpkg.com/babel-standalone@6/babel.min.js. This is a simple way to embed it on a webpage without having to do any other setup.
-- Install via Bower: `bower install babel-standalone`
-- Install via NPM: `npm install --save babel-standalone`
+- Use it via UNPKG: https://unpkg.com/@babel/standalone@6/babel.min.js. This is a simple way to embed it on a webpage without having to do any other setup.
```
Comment (Daniel15) [inline packages/babel-standalone/README.md]: We need to double check that this is the correct syntax for Unpkg URLs.

File: packages/babel-standalone/README.md [hunk 4]
```diff
@@ -16,11 +16,11 @@ It's true that using Babel through Webpack, Browserify or Gulp should be suffici
 Installation
 ============
 
-There are several ways to get a copy of babel-standalone. Pick whichever one you like:
+There are several ways to get a copy of @babel/standalone. Pick whichever one you like:
```
Comment (Daniel15) [inline packages/babel-standalone/README.md]: I think some of this text should still say `babel-standalone`, what do you think?

File: packages/babel-standalone/README.md [hunk 5]
```diff
@@ -16,11 +16,11 @@ It's true that using Babel through Webpack, Browserify or Gulp should be suffici
...
 
-- Use it via UNPKG: https://unpkg.com/babel-standalone@6/babel.min.js. This is a simple way to embed it on a webpage without having to do any other setup.
-- Install via Bower: `bower install babel-standalone`
-- Install via NPM: `npm install --save babel-standalone`
+- Use it via UNPKG: https://unpkg.com/@babel/standalone@6/babel.min.js. This is a simple way to embed it on a webpage without having to do any other setup.
+- Install via Bower: `bower install @babel/standalone`
```
Comment (Daniel15) [inline packages/babel-standalone/README.md]: This is definitely not right, but maybe we should just remove this line now :P

File: packages/babel-standalone/README.md (hunk 3 above)
Comment (Andarist) [reply packages/babel-standalone/README.md]: it is - syntax is as follows: `https://unpkg.com/package_name@version/path/to/file.js`

where `package_name` might be scoped

File: packages/babel-standalone/README.md (hunk 4 above)
Comment (Andarist) [reply packages/babel-standalone/README.md]: maybe just "standalone Babel"? 🤔

File: packages/babel-standalone/README.md (hunk 5 above)
Comment (xtuc) [reply packages/babel-standalone/README.md]: I'm +1 to drop Bower

**Pull request merged**